src/parser/jobs/mch/modules/GeneralCDDowntime.tsx

```diff
@@ -0,0 +1,47 @@
+import {Trans} from '@lingui/react'
+import ACTIONS from 'data/ACTIONS'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React from 'react'
+
+// Strict downtime allowance for drill/bio blaster/air anchor
+const DOWNTIME_ALLOWED_GCD = 100
+
+export default class GeneralCDDowntime extends CooldownDowntime {
+	trackedCds = [ {
+		cooldowns: [ACTIONS.WILDFIRE],
+		firstUseOffset: 10000,
+	}, {
+		cooldowns: [ACTIONS.BARREL_STABILIZER],
+		firstUseOffset: 3000,
+	}, {
+		cooldowns: [ACTIONS.REASSEMBLE],
+		allowedAverageDowntime: 5000,
+		firstUseOffset: 0,
+	}, {
+		cooldowns: [ACTIONS.AIR_ANCHOR],
+		allowedAverageDowntime: DOWNTIME_ALLOWED_GCD,
+		firstUseOffset: 9000,
+		// isAffectedBySpeed: true,
+	}, {
+		cooldowns: [
+			ACTIONS.DRILL,
+			ACTIONS.BIOBLASTER,
+		],
+		allowedAverageDowntime: DOWNTIME_ALLOWED_GCD,
+		firstUseOffset: 0,
+		// isAffectedBySpeed: true,
+	}, {
+		cooldowns: [ACTIONS.GAUSS_ROUND],
+		firstUseOffset: 3000,
+		resetBy: {actions: [ACTIONS.HEAT_BLAST], refundAmount: 15000},
+	}, {
+		cooldowns: [ACTIONS.RICOCHET],
+		firstUseOffset: 3000,
+		resetBy: {actions: [ACTIONS.HEAT_BLAST], refundAmount: 15000},
+	}]
+
+	checklistName = <Trans id="mch.cooldownDowntime.name">Use your cooldowns</Trans>
+	checklistDescription = <Trans id="mch.cooldownDowntime.suggestion">
+		Always make sure to use your abilities when they are available, but do not clip or delay your GCD to use them.
+	</Trans>
```

Comment:
Any particular reason for the overrides here? Personal preference would be to use the default ones defined in the core module (which are nearly the same, just with "OGCDs" where you have "cooldowns") to be more consistent with other jobs.

Reply:
Like SMN, these aren't all oGCDs (Air Anchor/Drill).  We actually had a user point this out for SMN, which lead to the change.  

If the goal is to have MCH use the actual same one as SMN, without making the translators translate it twice, I think you can say

`<Trans` id="smn.cooldownDowntime.name" />`

(empty tag with no value, likewise for description) and the same English and translations should be used.

Reply:
@KelosPelmand since there's at least 2 jobs now with GCD cooldowns using this module, maybe it would be worthwhile to generalize the default text?

Reply:
@KelosPelmand since there are now at least 2 jobs using this module with cooldowns on the GCD, maybe it's best to update the default text? Or maybe a flag for oGCDs vs cooldowns if we really want to keep that separation?